<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WSAMZN-1OLKJPF2" failures="0" tests="1" name="com.gapi.testCases.CreatePermissionGroup" time="0.864" errors="1" timestamp="2022-06-20T08:20:58 GMT" skipped="0">
  <testcase classname="com.gapi.testCases.CreatePermissionGroup" name="CreatePermissions" time="0.864">
    <error message="
[public void com.gapi.testCases.CreatePermissionGroup.CreatePermissions(java.util.Map,java.lang.String) throws java.lang.InterruptedException,java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: CreatePermissions([Parameter{index=0, type=java.util.Map, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.util.HashMap) {=, lastName=, accountName=, referenceTicket=, description=, executionFlag=Y, TestCaseName=Validate if we are able to create Permissions, title=, search_query=, yardi_id=, assetId=, customerReference=, action=automationPermissionAction, ExpectedStatusMessage=Created, detailedInstruction=, trackingNumber=, email=, isLoadingBayRequired=, permissionGroupId=dummyPG, comments=, requestType=, expectedStatusCode=201, ci=, errorMessage=, methodName=CreatePermissions, errorMessageType=, filters=, report_type=, estimatedDeliveryDate=, firstName=, site=, phone=, courier=, user_id=, name=automationPermissionName, assetName=, location=, notificationRecipients=, packageCount=, position=, page=, asset=, category=}]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.gapi.testCases.CreatePermissionGroup.CreatePermissions(java.util.Map,java.lang.String) throws java.lang.InterruptedException,java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: CreatePermissions([Parameter{index=0, type=java.util.Map, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.util.HashMap) {=, lastName=, accountName=, referenceTicket=, description=, executionFlag=Y, TestCaseName=Validate if we are able to create Permissions, title=, search_query=, yardi_id=, assetId=, customerReference=, action=automationPermissionAction, ExpectedStatusMessage=Created, detailedInstruction=, trackingNumber=, email=, isLoadingBayRequired=, permissionGroupId=dummyPG, comments=, requestType=, expectedStatusCode=201, ci=, errorMessage=, methodName=CreatePermissions, errorMessageType=, filters=, report_type=, estimatedDeliveryDate=, firstName=, site=, phone=, courier=, user_id=, name=automationPermissionName, assetName=, location=, notificationRecipients=, packageCount=, position=, page=, asset=, category=}]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CreatePermissions -->
  <system-out/>
</testsuite> <!-- com.gapi.testCases.CreatePermissionGroup -->
